### PR TITLE
Merge the big boi to production

### PR DESCRIPTION
**Pull request by JimmyEllercamp**
Dis be my awesome game plez review and get back to me

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,57 @@
...
+
+        axios({
+                url: apiUrl,
+                method: 'GET',
+                dataResponse: 'json',
+                params: {
+                    format: 'json',
+                    count: 1,
+                },
+            }).then( (response) => {
```
Comment (Kamlesh-62) [inline src/App.js:27]: Everything looks good, Your code is dry and easy to understand. Very well organized. 👏🏻

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,57 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
```
Comment (Kamlesh-62) [inline src/App.js:50]: I love the way you have created all sections in components. so it's easy to review your code. Great work..👍🏻

File: src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// Footer.js
+
```
Comment (Kamlesh-62) [inline src/components/Footer.js:2]: Footer looks good 👌🏻

File: src/components/GameCard.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+// GameCard.js
+import { useEffect, useState } from 'react';
+
+const GameCard = (props) => {
+
+    const {question, answer, value, handleClick, reset, readyState} = props;
```
Comment (Kamlesh-62) [inline src/components/GameCard.js:6]: destructuring Props is good practice to not repeat props all time, while assigning props value. Great work 👏🏻👏🏻

File: src/components/Teams.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+// Teams.js
```
Comment (Kamlesh-62) [inline src/components/Teams.js:1]: very welled organized code. But if you would add some comment. so it would definitely help reviewer. 👍🏻  👏🏻

Comment (JimmyEllercamp) [reply src/components/Teams.js:1]: YES I am so sorry, I never considered anyone else would be reading this code!! I have absolutely learned for next time

Comment (JimmyEllercamp) [reply src/components/Teams.js:1]: Thanks mate!

**Pull request merged**